bench/Main.hs

```diff
@@ -29,11 +29,6 @@ main :: IO ()
 main = defaultMain
   [ bgroup "micro" micros ]
 
-#if !(MIN_VERSION_deepseq(1,4,2))
```

Comment:
We require `base >= 4.11` and https://hackage.haskell.org/package/deepseq-1.4.2.0 only supports `< 4.10` so we always use newer versions